plugins/services/src/js/reducers/JSONParserReducers.js

```diff
@@ -10,13 +10,23 @@ import {JSONParser as portDefinitions} from './serviceForm/PortDefinitions';
 import {JSONParser as portMappings} from './serviceForm/PortMappings';
 import {JSONParser as network} from './serviceForm/Network';
 import {simpleParser} from '../../../../../src/js/utils/ParserUtil';
+import {findNestedPropertyInObject} from '../../../../../src/js/utils/Util';
+import Transaction from '../../../../../src/js/structs/Transaction';
 
 const {MESOS, DOCKER} = ContainerConstants.type;
 
 module.exports = [
   simpleParser(['id']),
   simpleParser(['instances']),
-  simpleParser(['container', 'type']),
+  function (state) {
```

Comment:
Wonder if should move this one as well as all the `container` parser to dedicated file so that they are properly grouped...